README.md

```diff
@@ -295,24 +303,26 @@ constructor. The event is not handled under the following conditions:
 pages](https://mathiasbynens.github.io/rel-noopener/).
 
 ### Navigating programmatically
-To can navigate routes you can emit `'pushState'`, `'popState'` or
-`'replaceState'`. See [#events](#events) for more details about these events.
+
+To navigate routes, `'pushState'`, `'popState'` or
+`'replaceState'` can be emitted. See [#events](#events) for details.
 
 ## Server Rendering
+
 Choo was built with Node in mind. To render on the server call
 `.toString(route, [state])` on your application.
 
 ```js
-var html = require('choo/html')
-var choo = require('choo')
+const html = require('choo/html')
+const choo = require('choo')
```

Comment:
The use of `var` instead of `const` has been the house style on this and related projects for quite a while. See for example: https://github.com/yoshuawuyts/sheet-router/issues/78. There should an issue on one of the submodules that describes the reasoning, but I can't remember where exactly to find it, maybe someone else can.